src/h5web/visualizations/heatmap/ColorMapSelector.tsx

```diff
@@ -27,29 +55,45 @@ function ColorMapSelector(): JSX.Element {
     shallow,
   ]);
 
-  const gradientStyles = {
+  const customStyles = {
     option: (
-      selectStyles: CSSProperties,
-      { data: { label } }: { data: { label: ColorMap } }
+      provided: CSSProperties,
+      { isSelected }: { isSelected: boolean }
+    ) =>
+      isSelected
+        ? {
+            border: 'solid 1px #1b998b',
+          }
+        : {},
+    singleValue: (
+      provided: CSSProperties,
+      { data: { value } }: { data: { value: ColorMap } }
     ) => ({
-      ...selectStyles,
-      backgroundImage: generateCSSLinearGradient(INTERPOLATORS[label], 'right'),
-      marginTop: '1px',
-      marginBottom: '1px',
+      ...provided,
+      backgroundImage: generateCSSLinearGradient(INTERPOLATORS[value], 'right'),
+      padding: '0.25rem',
+      color: 'white',
+      width: '100%',
```

Comment:
Even if static, these styles must be inline otherwise they will be overwritten by the provided styles